src/main/java/org/robolancers321/Constants.java

```diff
@@ -17,23 +17,23 @@
 public final class Constants {
   public static class OperatorConstants {
     public static final int kDriverControllerPort = 0;
+    public static final int kManipulatorControllerPort = 1;
   }
 
   public static final class Arm {
     public static final class Anchor {
-      public static final int kAnchorPort = 0;
-      public static final int kAnchorEncoderPort = 0;
+      public static final int kAnchorPort = 15;
 
       public static final boolean kInverted = false;
-      public static final double kAnchorLength = 0;
+      public static final double kAnchorLength = 33; // in
       public static final double kZeroPosition = 0;
       public static final double kMinAngle = Double.NEGATIVE_INFINITY;
       public static final double kMaxAngle = Double.POSITIVE_INFINITY;
```

Comment:
Why infinity?

Reply:
Have to tune the soft limit. Infinity is the abr value, prob should've changed it